### PR TITLE
Jacob armiger/muse 20 contact form component

### PR DESCRIPTION
**Pull request by jacob-armiger**
### Jira Issue ID(s)
20
### Changes
**Contact Form**
Creates the necessary fields for the contact form and styles the page

**nodemailer**
Sets up nodemailer so that emails can be sent to Muse when a user submits the form. A username and password for the nodemailer transport will need to be provided inside the `.env` file

### CODE REVIEW

File: pages/contact.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { NextPage } from "next";
+import ContactForm from "components/ContactForm";
+const contact: NextPage = () => {
+  return <ContactForm></ContactForm>;
```
Comment (tbatk) [inline pages/contact.tsx]: The Contact Form component needs to be placed within the layout component here.

**Review by tbatk — CHANGES_REQUESTED** [review]
Looks good! The only change I think that needs to be made is on the contact page. Once that is fixed, you'll be good to go.

File: pages/contact.tsx (hunk 1 above)
Comment (jacob-armiger) [reply pages/contact.tsx]: I've added the layout component, but need to fix some styling before committing this PR. The header/footer cuts off the contact form. Will commit a fix soon!

**Review by rluberto — APPROVED** [review]
Everything looks great. Thanks for making the changes!

**Comment by jacob-armiger** [comment]
I have committed the styling fix and resolved merge conflicts. Please look over these and let me know if I need to make any changes. Thanks!

**Review by tbatk — APPROVED** [review]

**Pull request merged**